### DISCUSSION TITLE
Prediction

### DISCUSSION THREAD

**Discussion by SardanDev** [Chatterino/chatterino2#4439] [Q&A] [answered]
Hello, could you please add a way to launch prediction please

**Comment by Felanbird** [comment, marked as answer]
We* cannot, Twitch does not supply us with an API that we can properly utilize to implement predictions into Chatterino

cc: https://github.com/Chatterino/chatterino2/issues/2740

**Reply by SardanDev** [reply]
Oh, too bad and differents langages support ?

**Reply by Felanbird** [reply]
> Oh, too bad and differents langages support ?

If you mean settings translations, it's been discussed setting up Crowdin for translations, but surprisingly all the main Chatterino people who speak another language, seem to not care having to use English for Chatterino, so the addition hasn't made it past discussion.

If you meant support in the sense of RTL languages, basic support was added in 2.4.0 https://github.com/Chatterino/chatterino2/pull/3958

**Reply by SardanDev** [reply]
> > Oh, too bad and differents langages support ?
> 
> If you mean settings translations, it's been discussed setting up Crowdin for translations, but surprisingly all the main Chatterino people who speak another language, seem to not care having to use English for Chatterino, so the addition hasn't made it past discussion.
> 
> If you meant support in the sense of RTL languages, basic support was added in 2.4.0 #3958

Ok i understand but it's better when people who doesn't speak or understand english can use it :)

**Reply by SardanDev** [reply]
And a a way to had the channel points in a widget ?

**Reply by Felanbird** [reply]
There's no API for channel point total either.

**Reply by SardanDev** [reply]
But i see in the network part the request use by Twitch for the prediction

**Reply by comradekingu** [reply]
@Invincible812 
Happy to help if that solution is avoided.
https://hosted.weblate.org/projects/gumbo/#information and https://hosted.weblate.org/projects/twire/#information
are already on Weblate, and that seems to work fine. :)

**Reply by pajlada** [reply]
@Invincible812 To clarify: We're fine with adding & supporting translations, but none of the current contributors are making it their priority because it's a pretty big initiative. We would be happy to have a new contributor come in and add translation support.

Generally it's best to keep discussions & issues related to a single topic only so threads don't get muddled up with multiple topics - if you're interested in tracking it feel free to create a new discussion for it.

**Comment by Mm2PL** [comment]
10 mar 2023 16:54:37 Sardan ***@***.***>:

> But i see in the network part the request use by Twitch for the prediction


If we could use whatever the website uses, we would have added this functionality basically when it was created. But we can't really do that, it's impractical and probably breaks ToS/the developer agreement. However the future I predict is that we will be forced to use these APIs. Helix is slowly replacing everything, except that most of the endpoints are crippled.

What chatterino uses, is a readonly channel points pubsub topic.
